### PR TITLE
sdxl: configure edm inference and dynamic thresholding

### PR DESCRIPTION
**Pull request by Fzilan**
# What does this PR do?

####  1 Support args setting of EDMDiscretization in SDXL inference.

Usage of EDMDiscretization: 

=======**Example ① EDMEulerScheduler**=======
directly set the following args in `sampling_without_streamlit.py`:
```shell
--sampler EulerEDMSampler
--discretization EDMDiscretization 
--sigma_min 0.002
--sigma_max 80.0
--rho 7.0
```

**or** 

set `--sampler EulerEDMSampler` in `sampling_without_streamlit.py`
and set discretization_config as follows in denoiser_config of `examples/stable_diffusion_xl/configs/inference/sd_xl_base.yaml`
```shell
...
        discretization_config:
          target: gm.modules.diffusionmodules.discretizer.EDMDiscretization
          params:
            sigma_min: 0.02
            sigma_max: 80.0
            rho: 7.0
```

=======**Example ② EDMDPMSolverMultistepScheduler**=======
directly  set the following args in `sampling_without_streamlit.py`:
```shell
--sampler DPMPP2MSampler  #dpm++ 2nd order solver, multistep
--discretization EDMDiscretization 
--sigma_min 0.002
--sigma_max 80.0
--rho 7.0
```
**or** 

set `--sampler DPMPP2MSampler`  in `sampling_without_streamlit.py`
and set discretization_config as follows in denoiser_config of `examples/stable_diffusion_xl/configs/inference/sd_xl_base.yaml`
```shell
...
        discretization_config:
          target: gm.modules.diffusionmodules.discretizer.EDMDiscretization
          params:
            sigma_min: 0.02
            sigma_max: 80.0
            rho: 7.0
```
####  2 Support dynamic thresholding method for CFG (classifier-free guidance)
Useage:
```shell
python sampling_without_streamlit.py \
...
--thresholding True 
--dynamic_thresholding_ratio 0.995 
--sample_max_value 1.0
... 
```

## fixs 
1. dreambooth sdxl config updates
2. timestepbias bug

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/stable_diffusion_xl/demo/sampling_without_streamlit.py [hunk 1]
```diff
@@ -318,6 +379,14 @@ def sample(args):
 
     task = args.task
 
+    if args.discretization is None:
+        assert "discretization_config" in config.model.params.denoiser_config.params
+        args.discretization = config.model.params.denoiser_config.params.discretization_config
+    else:
+        config.model.params.denoiser_config.params.discretization_config = get_discretization(
+            args.discretization, args.sigma_min, args.sigma_max, args.rho
+        )
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/demo/sampling_without_streamlit.py:388]: 两个分支加下打印

**Review by zhanghuiyao — APPROVED** [review]

File: examples/stable_diffusion_xl/demo/sampling_without_streamlit.py [hunk 2]
```diff
@@ -14,7 +14,15 @@
     os.environ["MS_PYNATIVE_GE"] = "1"
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/demo/sampling_without_streamlit.py:14]: EDMEulerScheduler/EDMDPMSolverMultistepScheduler sampler 增加readme或yaml

Comment (Fzilan) [reply examples/stable_diffusion_xl/demo/sampling_without_streamlit.py:14]: done

File: examples/stable_diffusion_xl/demo/sampling_without_streamlit.py (hunk 1 above)
Comment (Fzilan) [reply examples/stable_diffusion_xl/demo/sampling_without_streamlit.py]: done

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**